src/components/tasks/card/index.tsx

```diff
@@ -369,6 +375,16 @@ const Card: FC<Props> = ({
         );
     };
 
+    const handleAssigneeName = (e: any) => {
+        setAssigneeName(e.target.value);
+        setShowSuggestion(true);
+    };
```

Comment:
use proper type please, stop using any

Reply:
@Pratiyushkumar sorry for using any.
This is my first time working with typescript, i did try with different types to make this work, but i was not able to so i used type any.
I will rectify this

Reply:
fixed @Pratiyushkumar